### PR TITLE
[3110] Correct error page styling

### PR DESCRIPTION
**Pull request by faissalMT**
### Context
The error page did not have the correct styling or content, it should have looked like: https://www.find-postgraduate-teacher-training.service.gov.uk/xxx

### Changes proposed in this pull request
Correct the styling

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally
- [ ] Product review

### CODE REVIEW

File: app/views/errors/not_found.html.erb [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+<div class="govuk-width-container">
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">The page you were looking for doesn't exist.</h1>
-      <p class="govuk-body">You may have mistyped the address or the page may have moved.</p>
-      <p class="govuk-body">If you are the application owner check the logs for more information.</p>
+      <h1 class="govuk-heading-xl">Page not found</h1>
+      <p class="govuk-body">If you entered a web address please check it was correct.</p>
+      <p class="govuk-body">
+        <%= mail_to(Settings.service_support.contact_email_address) do %>
```
Comment (aliuk2012) [inline app/views/errors/not_found.html.erb]: need to add `govuk-link` css class

File: app/views/errors/internal_server_error.html.erb [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
   <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <h1 class="govuk-heading-xl">Sorry, something went wrong</h1>
+    </div>
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">We're sorry, but something went wrong.</h1>
-      <p class="govuk-body">If you are the application owner check the logs for more information.</p>
+      <p class="govuk-body">Sorry, we're experiencing technical difficulties.</p>
+      <p class="govuk-body">
+        <%= mail_to(Settings.service_support.contact_email_address) do %>
```
Comment (aliuk2012) [inline app/views/errors/internal_server_error.html.erb]: need to add `govuk-link` css class

File: config/application.rb [hunk 3]
```diff
@@ -23,6 +23,6 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
 
-    config.exceptions_app = routes
+    config.exceptions_app = self.routes
```
Comment (aliuk2012) [inline config/application.rb]: why was this change needed?

Comment (faissalMT) [reply config/application.rb]: It wasn't, I was just trying to get the custom error pages to show instead of the debug error pages

File: app/views/errors/internal_server_error.html.erb [hunk 4]
```diff
@@ -1,8 +1,16 @@
-<main role="main" class="govuk-main-wrapper" id="main-content">
```
Comment (aliuk2012) [inline app/views/errors/internal_server_error.html.erb]: Why did you have to remove `main` element? Was it because these should be partials and therefore should be using the main application layout?

File: config/application.rb (hunk 3 above)
Comment (aliuk2012) [reply config/application.rb]: Thanks for explaining, I've not come across it before which is why I was asking 😄

File: app/views/errors/internal_server_error.html.erb (hunk 4 above)
Comment (faissalMT) [reply app/views/errors/internal_server_error.html.erb]: Because the application template was already inserting a `<main>`, which means you'd end up with a duplicate `<main>`.

Comment (aliuk2012) [reply app/views/errors/internal_server_error.html.erb]: Thank you for clarifying

File: config/environments/development.rb [hunk 5]
```diff
@@ -12,7 +12,7 @@
   config.eager_load = false
 
   # Show full error reports.
-  config.consider_all_requests_local = true
+  #config.consider_all_requests_local = true
```
Comment (dankmitchell) [inline config/environments/development.rb]: I wouldn't do this.

Comment (faissalMT) [reply config/environments/development.rb]: Oops, I forgot to put that back

**Comment by aliuk2012** [comment]
[Error pages looks wrong](https://trello.com/c/yRCoEocg/3110-error-pages-looks-wrong)

File: app/views/errors/unprocessable_entity.html.erb [hunk 6]
```diff
@@ -1,9 +1,9 @@
-<main role="main" class="govuk-main-wrapper" id="main-content">
+<div class="govuk-width-container">
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <h1 class="govuk-heading-xl">The change you wanted was rejected.</h1>
       <p class="govuk-body">Maybe you tried to change something you didn't have access to.</p>
       <p class="govuk-body">If you are the application owner check the logs for more information.</p>
```
Comment (defong) [inline app/views/errors/unprocessable_entity.html.erb]: need content for that

Comment (faissalMT) [reply app/views/errors/unprocessable_entity.html.erb]: I'm not sure what you mean

Comment (defong) [reply app/views/errors/unprocessable_entity.html.erb]: So a candidate will be reading

```
The change you wanted was rejected.
Maybe you tried to change something you didn't have access to.
If you are the application owner check the logs for more information.
```

Comment (faissalMT) [reply app/views/errors/unprocessable_entity.html.erb]: Will make a card for this
